packages/hub/src/scope.ts

```diff
@@ -326,6 +337,7 @@ export class Scope implements ScopeInterface {
     this._extra = {};
     this._user = {};
     this._contexts = {};
+    this._requestSessionStatus = undefined;
```

Comment:
Same here, for consistency sake keep this after `fingerprint`, so at least it is easier to scan over these with eyes and compare / spot differences.